File: Python/D6 - T/3 - how many digits in num/3.py
# Write a python program to print the length of the given number.
#  for example: input = 2005, ans = 4

num = int(input('Enter the number : '))
digit = 0
count = 0

while(num != 0):
    digit = num % 10
    count = count + 1
    num = num // 10

print(count)

File: C/D4/3 - sum of series3/3.cpp
// Write a C program to find the total of numbers upto given by user in below series manner.
// 1/1 - 1/2 + 1/3 - 1/4 + 1/5 - .....

#include <stdio.h>
#include <math.h>

int main(){
    float ans = 0.0;
    int i, num;

    printf("Enter a number : ");
    scanf("%d", &num);

    for(i = 1; i <= num; i++){
        ans = (i % 2 != 0) ? ans + (1.0/i) : ans - (1.0/i);
    }

    printf("Total of the series is %f", ans);
}